src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShardSyncer.java

```diff
@@ -1,843 +1,24 @@
-/*
```

Comment:
Reintroduce the copyright licenses. All files need it doesn't matter what type of file it is.